google/cloud/spanner/value.h

```diff
@@ -305,6 +294,18 @@ class Value {
   google::protobuf::Value value_;
 };
 
+/**
+ * Factory to construct a "null" Value of the specified type `T`.
+ *
+ * This is equivalent to passing to the constructor an `optional<T>` without a
```

Comment:
Perhaps "equivalent to passing \<thing\> to the constructor" is clearer than "equivalent to passing to the constructor \<thing\>".